theme/themes/eea/extras/header.less

```diff
@@ -545,6 +555,15 @@
 }
 
 @media only screen and (min-width: @computerBreakpoint) {
+  .eea.header .subsite-logo {
+    position: absolute;
+    z-index: 1;
+    top: 50%;
+    left: calc(100% + 30px);
```

Comment:
@kreafox 30px should be a variable I think that it's like the gap value
so I would add that as a variable and reference it here